FF1Lib/FF1Rom.cs

```diff
@@ -1183,7 +1203,13 @@ public void Randomize(Blob seed, Flags flags, Preferences preferences)
 
 			if (flags.TournamentSafe || preferences.CropScreen) ActivateCropScreen();
 
-			WriteSeedAndFlags(seed.ToHex(), Flags.EncodeFlagsText(flags));
+			String seedHex = seed.ToHex();
+			String encodedFlagText = Flags.EncodeFlagsText(flags);
+			WriteSeedAndFlags(seedHex, encodedFlagText);
+
+			Blob romStampPayload = Encoding.ASCII.GetBytes(System.Text.Json.JsonSerializer.Serialize(new RandomizerDigest(seedHex, encodedFlagText, FFRVersion.Sha, inputRomHash)));
```

Comment:
Did you see my earlier comment about a null terminator? I think you achieve this, at least, by doing `PutInBank(0x1B, 0xA000 + romStampPayload.size(), Blob.FromHex("0x00"));` or something like that. Then you can scan for the null byte which will be a lot better than scanning for a "}"

Reply:
Also something like jsonAscii or jsonBlob is just a better name for this in my opinion, but that's _very_ nitpicky and it's no big deal.

Reply:
done and done